config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -104,7 +107,6 @@ presubmits:
     - release-\d+\.\d+ # per-release settings
     labels:
       preset-dind-enabled: "true"
-      preset-kind-volume-mounts: "true"
```

Comment:
this is on the wrong job
also testing the ipv6 job is slow because the CPU is under-allocated for building vs the ipv4 job
https://github.com/kubernetes/test-infra/pull/34840